File: 241027/a부터 b까지의 합/sum-from-a-to-b.cpp
#include <iostream>
using namespace std;

int main() {
    int a, b;
    cin >> a >> b;

    int sum_val = 0;

    for(int i = a; i <= b; i++) {
        sum_val += i;
    }

    cout << sum_val;
    return 0;
}

File: 241027/a부터 b까지의 합/sum-from-a-to-b.js
const fs = require("fs");
let input = fs.readFileSync(0).toString().trim().split(" ");

let a = Number(input[0]);
let b = Number(input[1]);
let sumVal = 0;

for (let i = a; i <= b; i++) {
    sumVal += i;
}

console.log(sumVal);